core/rovecomm.py

```diff
@@ -73,6 +86,34 @@ def print(self):
         print('----------')
 
 
+class RoveCommCollection:
+    '''
+    Creates a separate thread to read all RoveComm connections
+    '''
+    connections = []
+
+    def __init__(self):
+        self.thread = threading.Thread(target=self.listen)
+        self.thread.daemon = True
```

Comment:
If the program terminates abnormally, the daemon thread may terminate without calling cleanup function

Reply:
This is a good point, but it might not be as important in this case as in some other cases. If the program terminates abnormally we have a bigger issue than threads not cleaning up.